### PR TITLE
add h4 styling for product, technology and application

### LINKED ISSUE #718: [Product] style h4 titles
https://franklin.moleculardevices.com/products/cellular-imaging-systems/high-content-imaging/imagexpress-pico

H4 is not styled yet.

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #718

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-pico
- After: https://718-h4--moleculardevices--hlxsites.hlx.page/products/cellular-imaging-systems/high-content-imaging/imagexpress-pico

### CODE REVIEW

**Comment by mhaack** [comment]
@lydiapuric LGTM but do we have more test URLs, I see you change the styles for applications as well.

**Comment by lydiapuric** [comment]
TBH I was not able to find more product, application or technology  pages with h2 / h4 combination. However, I know, that they all 3 types follow in general same style. Do you want me to remove the style changes for technology/ application for now?

File: styles/styles.css [hunk 1]
```diff
@@ -1527,6 +1545,26 @@ main .section.background-molecules {
     margin-bottom: 10px;
 }
 
+.technology .section.tabs h2:has(+ h4),
```
[the remark this answers is not in the document]
Comment (lydiapuric) [reply styles/styles.css:1548]: It is more greedy but will be ok in this. Adjusting

**Review by duynguyen — APPROVED** [review]
LGTM. if there's no h4 in any other application or technology page, I would vote for removing it from the PR for now (to save a few bits for styles.css size)

**Pull request merged**